### PR TITLE
ADD appBarBreakpoint

### PR DESCRIPTION
**Pull request by wer-mathurin**
Thsi PR add the ability to display an AppBar on any desired Breakpoint.  It can be usefull to display an AppBar even on a desktop application. 

related to : https://github.com/flutter/flutter/issues/130117

### CODE REVIEW

File: packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart [hunk 1]
```diff
@@ -215,6 +216,13 @@ class AdaptiveScaffold extends StatefulWidget {
   /// Defaults to [Breakpoints.smallDesktop].
   final Breakpoint drawerBreakpoint;
 
+  /// Option to override the appBarBreakpoint for the usage of an AppBar when
+  /// needed that not depend on the drawer visibility
+  ///
+  /// By not providing a [Breakpoint] by default an [AppBar] will show on
+  /// [Breakpoints.smallDesktop] if the useDrawer is set to true
```
Comment (gspencergoog) [inline packages/flutter_adaptive_scaffold/lib/src/adaptive_scaffold.dart:223]:
```suggestion
  /// An optional [Breakpoint] which overrides the [appBar] breakpoint to display
  /// an [AppBar] without depending on the drawer visibility.
  ///
  /// By default, an [AppBar] will show on [Breakpoints.smallDesktop] if [useDrawer] is set
  /// to true.
```

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

**Comment by gspencergoog** [comment]
You will need to bump the version and add a change to the `CHANGELOG.md`

**Comment by wer-mathurin** [comment]
@gspencergoog 
> You will need to bump the version and add a change to the `CHANGELOG.md`

**DONE**

**Review by Piinks — APPROVED** [review]
Flutter_LGTM

**Pull request merged**